launcher/image/test/scripts/test_gpu_workload.sh

```diff
@@ -0,0 +1,17 @@
+ #!/bin/bash
+set -euo pipefail
+source util/read_serial.sh
+
+# This test requires the workload to run and printing
+# corresponding messages to the serial console.
+SERIAL_OUTPUT=$(read_serial $1 $2) 
+print_serial=false
+
+if echo $SERIAL_OUTPUT | grep -q 'TEST PASSED'
```

Comment:
where is this "TEST PASSED" coming from, from the workload?

Reply:
yes, it is coming from workload.